packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1883,6 +1889,35 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     }
   }
 
+  @override
+  TextInputConfiguration get textInputConfiguration {
+    return TextInputConfiguration(inputType: widget.keyboardType,
+      obscureText: widget.obscureText,
+      autocorrect: widget.autocorrect,
+      smartDashesType: widget.smartDashesType ?? (widget.obscureText ? SmartDashesType.disabled : SmartDashesType.enabled),
+      smartQuotesType: widget.smartQuotesType ?? (widget.obscureText ? SmartQuotesType.disabled : SmartQuotesType.enabled),
+      enableSuggestions: widget.enableSuggestions,
+      inputAction: widget.textInputAction ?? (widget.keyboardType == TextInputType.multiline
+        ? TextInputAction.newline
+        : TextInputAction.done
+      ),
+      textCapitalization: widget.textCapitalization,
+      keyboardAppearance: widget.keyboardAppearance,
+      autofillConfiguration: (widget.autofillHints?.isEmpty ?? true) ? null : AutofillConfiguration(
+        uniqueIdentifier: toStringShort(),
```

Comment:
Not sure if this is a good idea. I can use the hash code instead.

Reply:
It seems like `toStringShort` is hash code and `runtimeType`?  Maybe someone more familiar can say what's most appropriate here.

Reply:
Changed to hashCode. It seems in a release build everything is just called "Widget" if it doesn't have a key.